py-polars/tests/unit/datatypes/test_temporal.py

```diff
@@ -925,7 +931,7 @@ def test_groupby_dynamic_when_conversion_crosses_dates_7274() -> None:
     ).agg(pl.col("value").count())
     expected = pl.DataFrame({"timestamp": [datetime(1970, 1, 1)], "value": [2]})
     expected = expected.with_columns(
-        pl.col("timestamp").dt.replace_time_zone("+01:00"),
+        pl.col("timestamp").dt.replace_time_zone("Africa/Lagos"),
```

Comment:
putting 'Africa/Lagos' instead of '+01:00' to avoid the warning (and because the purpose of this test is to test `groupby_dynamic`, not `replace_time_zone`)